resources/Materials/TestSuite/stdlib/upgrade/1_37_to_1_38.mtlx

```diff
@@ -23,4 +23,18 @@
          <parameter name="axis" type="vector3" value="0.0, 0.0, 1.0"/>
       </rotate3d>
    </nodegraph>
+
+   <nodedef name="ND_Test" node="test">
+      <input name="add" type="float" />
+   </nodedef>
+   <nodegraph name="NG_Test" nodedef="ND_Test">
+      <add name="add" type="float">
```

Comment:
For completeness, it would be good to have an input and an output child under the nodegraph with duplicate names. Maybe as a second nodegraph to check.